go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/google/gofuzz v1.1.0 // indirect
 	github.com/leodido/go-urn v1.2.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/serverlessworkflow/sdk-go v1.0.0 // indirect
```

Comment:
why import v1？

Reply:
This is happening in his branch, I made the same comment in the other PR.

Reply:
oh, that again. lol